Source/UI/NewUI/BillingForm/View/BillingFormPhoneNumberText.swift

```diff
@@ -1,101 +1,31 @@
+import Checkout
 import UIKit
 import PhoneNumberKit
 
 protocol BillingFormPhoneNumberTextDelegate {
-    func updateCountryCode(code:Int)
-    func validateNumber()
+    func phoneNumberIsUpdated(number: String?)
 }
 
-final class BillingFormPhoneNumberText: BillingFormTextField {
-
-    // MARK: - Properties
+final class BillingFormPhoneNumberText: PhoneNumberTextField, BillingFormTextField {
 
+    var type: BillingFormCell?
     var phoneNumberTextDelegate: BillingFormPhoneNumberTextDelegate?
-    var phoneNumber: PhoneNumber?
-    private let phoneNumberKit = PhoneNumberKit()
-
-    var partialFormatter: PartialFormatter {
-       PartialFormatter(phoneNumberKit: phoneNumberKit, defaultRegion: "GB", withPrefix: true)
-    }
-
-    /// National Number (e.g. )
-    var nationalNumber: String {
-        let rawNumber = text ?? ""
-        return partialFormatter.nationalNumber(from: rawNumber)
-    }
-
-    var isValidNumber: Bool {
-        let rawNumber = text ?? ""
-        do {
-            phoneNumber = try phoneNumberKit.parse(rawNumber.replacingOccurrences(of: " ", with: ""))
-            return true
-        } catch {
-            return false
-        }
-    }
-
-    private var previousTextCount = 0
-    private var previousFormat = ""
-    private lazy var _defaultRegion: String = PhoneNumberKit.defaultRegionCode()
 
     init(type: BillingFormCell?, tag: Int, phoneNumberTextDelegate: BillingFormPhoneNumberTextDelegate) {
-        super.init(type: type,tag: tag)
+        self.type = type
+        super.init(frame: .zero)
+        self.tag = tag
+        self.withPrefix = true
```

Comment:
this `withPrefix` is not used here? Is it used somewhere else, can this just be a default value?